Makefile

```diff
@@ -45,14 +45,28 @@ lint:
 
 .PHONY: security
 security:
-	bandit -s B101 -r aea packages scripts
-	bandit -s B101 -r tests
-	bandit -s B101 -r benchmark
-	safety check
+	bandit -s B101 -r aea benchmark examples packages scripts tests
+	safety check -i 37524 -i 38038 -i 37776 -i 38039
 
 .PHONY: static
 static:
-	mypy aea benchmark packages tests scripts
+	mypy aea benchmark examples packages scripts tests
+
+.PHONY: misc_checks
+misc_checks:
+	# pip install '.[all]'
+	# python scripts/freeze_dependencies.py -o requirements.txt
+	# liccheck -s strategy.ini -r requirements.txt -l PARANOID
+	# rm -fr requirements.txt
```

Comment:
@marcofavorito I cannot get this to work locally..it claims due to editable ...

Reply:
`liccheck` works by checking the packages installed in the current environment.

You have to create a temporary virtual environment and install the `aea` package, like it's done in `tox -e liccheck`.

Reply:
@DavidMinarsch 
do we really need to check tests with mypy?

Reply:
Yes 100% mypy everywhere.